### PR TITLE
updates

### PR DESCRIPTION
**Pull request by realthreeseconds**

### CODE REVIEW

File: src/main/java/forceitembattle/commands/player/trade/CommandTrade.java
Comment (McPlayHD) [inline src/main/java/forceitembattle/commands/player/trade/CommandTrade.java:1]: What's this class doing?

File: src/main/java/forceitembattle/manager/TradingManager.java [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+    public boolean hasInvite(ForceItemPlayer player) {
+        return this.pendingTradeRequests.containsKey(player);
+    }
+
+    public void sendTradeRequest(ForceItemPlayer player, ForceItemPlayer target) {
+        if(this.hasInvite(target)) {
+            //already got an invite
+            return;
+        }
+
```
Comment (McPlayHD) [inline src/main/java/forceitembattle/manager/TradingManager.java:38]: This is not finished yet?

**Review by McPlayHD — APPROVED** [review]
Is the CommandTrade empty on purpose? And I guess the TradingManager is also not finished yet?

**Comment by realthreeseconds** [comment]
> Is the CommandTrade empty on purpose? And I guess the TradingManager is also not finished yet?

Yes its empty on purpose, so its not finished yet

**Pull request merged**